easybuild/easyblocks/c/cuda.py

```diff
@@ -243,6 +247,18 @@ def create_wrapper(wrapper_name, wrapper_comp):
         # Also create the lib dir as a symlink
         symlink('lib64', os.path.join(new_stubs_dir, 'lib'), use_abspath_source=False)
 
+        # Packages like xpra look for version independent pc files.
+        # See e.g. https://github.com/Xpra-org/xpra/blob/master/setup.py#L206
+        # Distros provide these files, so let's do it here too
+        pkgconfig_dir = os.path.join(self.installdir, 'pkgconfig')
+        pc_files = expand_glob_paths([os.path.join(pkgconfig_dir, '*.pc')])
```

Comment:
This doesn't always work apparently, fails with CUDA `10.1.105` because there are no pkgconfig files included at all:

```
No files found using glob pattern
'/user/gent/400/vsc40023/eb_arcaninescratch/CO7/skylake-ib/software/CUDA/10.1.105-GCC-8.2.0-2.31.1/pkgconfig/*.pc`
```

So this should be guarded with:

```python
if os.path.exists(pkgconfig_dir):
```

@damianam Does that make sense to you?

Reply:
None of the 10.x versions we have installed have a pkgconfig dir. So it's clearly needed

Reply:
Also the module generation needs to have this protected. Don't add nonexisting paths to PKG_CONFIG_PATH

Reply:
Makes sense. Update coming

Reply:
I pushed an update to address that. Note that I readded some of the pc files in the sanity check list. That's because those files are explicitly checked by other tools (and actually `nvjpeg.pc` is the whole reason for this PR), so I guess it makes sense to add them. Or are they not present in older CUDA releases?